zwave_js_server/client.py

```diff
@@ -184,11 +186,11 @@ async def connect(self) -> None:
         if self.version.max_schema_version < MAX_SERVER_SCHEMA_VERSION:
             self.schema_version = self.version.max_schema_version
 
-        self._logger.info(
+        LOGGER.info(
             "Connected to Home %s (Server %s, Driver %s, Using Schema %s)",
-            version.home_id,
-            version.server_version,
-            version.driver_version,
+            self.version.home_id,
```

Comment:
It's better to use a local variable if it's used more than once in the same scope. Attribute access is relatively more expensive.